### ISSUE TITLE
🐛 Bug Report: Cannot delete Project in Appwrite Cloud if signed in by github - requires password

### ISSUE THREAD

**Issue by dilberryhoundog** [appwrite/appwrite#5509]
### 👟 Reproduction steps

1. Sign in to appwrite cloud with github. 
2. Create a project.
3. Try to delete the project. 

This process uses a password to confirm deletion, but the account never persisted a password due to the github signin. I tried to use my github password but an error was thrown (invalid credentials)

### 👍 Expected behavior

not needed

### 👎 Actual Behavior

not needed

### 🎲 Appwrite version

Version 1.3.x

### 💻 Operating system

Linux

### 🧱 Your Environment

This was on appwrite cloud.

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by joeyouss** [comment]
Thank you for opening this. You need to enter a password but yes, with Github login, you do not have a password.
For now, you can do this:
- Go to account settings: https://cloud.appwrite.io/console/account
- Update password. For old one,put anything. For new one, set real password you want to use.
- Delete the project using the new password

**Comment by joeyouss** [comment]
Hi @dilberryhoundog if no further help is required here, can you please close the issue?

**Comment by subhendudash02** [comment]
> Thank you for opening this. You need to enter a password but yes, with Github login, you do not have a password. For now, you can do this:
> 
>     * Go to account settings: https://cloud.appwrite.io/console/account
> 
>     * Update password. For old one,put anything. For new one, set real password you want to use.
> 
>     * Delete the project using the new password

Yep, this worked for me. Projects can be deleted after resetting the password. Thanks @joeyouss.
But this workaround is really confusing. I think asking for verification in GitHub while deleting any project or changing any sensitive setting can be a good step, especially for those who signed up for appwrite using GitHub.

**Comment by joeyouss** [comment]
The PR is merged for this issue ! @dilberryhoundog : https://github.com/appwrite/appwrite/pull/5580

**Comment by sc231997** [comment]
Hi @joeyouss 
Any ETA when this fix will be available on appwrite cloud?

**Comment by joeyouss** [comment]
Hi, currently cloud is on 1.1.2 and we need the latest version (with the changes released) to access this. I am assuming it will take few more weeks to upgrade to newer versions.
